app/controllers/api/v2/hosts_controller.rb

```diff
@@ -123,7 +123,7 @@ def destroy
       eos
 
       def status
-        render :json => { :status => @host.host_status }.to_json if @host
+        render :json => { :status => @host.get_status('HostStatus::ConfigurationStatus').to_label }.to_json if @host
```

Comment:
Should we expand this to include more data about the detailed statuses, or create a new API?